### ISSUE TITLE
Should boot-cljs-devtools fail if dependencies are missing

### ISSUE THREAD

**Issue by arichiardi** [boot-clj/boot-cljs-devtools#8]
At the moment we throw a warning like:

```
You are missing necessary dependencies for boot-cljs-repl.
Please add the following dependencies to your project:
binaryage/dirac
```

Then the task chain moves forward, in case of missing dirac throwing an exception anyways:

```
java.io.FileNotFoundException: Could not locate dirac/nrepl__init.class or dirac/nrepl.clj on classpath.
```

Or a confusing:

`WARNING: preload namespace devtools.preload does not exist`

I wanted to brainstorm whether we should fail earlier instead of warn.

**Comment by martinklepsch** [comment]
Seems reasonable to me to throw an exception when dependencies are not met. 👍

**Comment by arichiardi** [comment]
Implemented in 22a80aa
